afctl/parser_helpers.py

```diff
@@ -46,7 +46,7 @@ def add_git_origin(files):
                 print("Updating git origin.")
                 Utility.update_config(files['project_name'], {'global':{'git':{'origin':origin}}})
                 print("Setting origin as : {}".format(origin))
-
+            print(colored("Set personal access token for Github. Run 'afctl config global -t <token>'", 'yellow'))
```

Comment:
can we configure python logger to print colored statements? based on info/warn/error/fatal etc.. 

would be better than using print also we can add some tags in the log like timestamp, filename, etc which can be used for debugging later.

Reply:
info will not get printed to console. I had added logger statements earlier